nautobot/dcim/tests/test_api.py

```diff
@@ -2218,8 +2243,8 @@ def setUp(self):
 
         super().setUp()
 
-        self.site1 = Site.objects.create(name="Test Site 1", slug="test-site-1")
-        self.site2 = Site.objects.create(name="Test Site 2", slug="test-site-2")
+        self.site1 = Site.objects.first()
+        self.site2 = Site.objects.last()
```

Comment:
```suggestion
        self.site = Site.objects.first()
```

`site2` isn't being used in this test